### PR TITLE
Example viz vignette 

### PR DESCRIPTION
**Pull request by wdwatkins**
This isn't finished yet, just thought I'd put my current progress up for comment.

### CODE REVIEW

File: vignettes/vizlab_example.Rmd [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  
+  This vignette will outline all the steps necessary to use the package, utilizing an example visualization that can be found on github at [https://github.com/USGS-VIZLAB/example](https://github.com/USGS-VIZLAB/example).  
+ 
+## Installation
+
+Vizlab can be downloaded from github via `devtools`:
```
Comment (jiwalker-usgs) [inline vignettes/vizlab_example.Rmd]: Should we get this up on GRAN soon?

File: vignettes/vizlab_example.Rmd [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  The `viz.yaml` file manages all the different components of the viz, and is used to generate the `make` files that build the final published product.  `viz.yaml` contains sections for each of the fetch, process, visualize, and publish steps, as well as general information about the viz.  Here we will examine the `viz.yaml` included in the example viz:
+
+### info
+```
+vizlab: "0.1.4"
+info:
+  id: example
+  name: Simple but complete vizualization
+  date: 2016-07-19
+  publish-date: 2016-08-02
```
Comment (jiwalker-usgs) [inline vignettes/vizlab_example.Rmd]: This is being used by new feature built by @lindsaycarr so should probably be called out as important.

File: vignettes/vizlab_example.Rmd [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  path: /example
+  analytics-id: UA-78530187-1
+  description: >-
+    This is meant to touch all features and act as an integration
+    test of the vizlab platform.
+```
+`analytics-id` contains the Google Analytics ID to be used.  The other fields here are self-explanatory and used to generate metadata for the viz.
```
Comment (jiwalker-usgs) [inline vignettes/vizlab_example.Rmd]: We will probably want to explain how to get a new GA id.

File: vignettes/vizlab_example.Rmd [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+### Setup
+
+The following steps are required to start a viz from scratch.  The first two have already been done for the example viz, so are not required here. These steps assume that the actual content creation phases are complete, *i.e.* there are scripts complete for data retrieval, processing, and figure creation.
+
+1. *Create the skeleton*: First the visualization directory structure should be created, using the `vizSkeleton` function.  This is unnecessary for the example viz, since the directories are already set up in the repo.  You can run this function in a dummy directory to see what happens.
+
+2. *Fill in the viz.yaml*: As described above, `viz.yaml` needs to be filled out in order to guide the creation of the various `make` files.  An empty skeleton `viz.yaml` is created by the `vizSkeleton` fuction.  The complete `viz.yaml` in the example viz repository can also be a useful reference. 
+
+3. *Create the profile*: Next, run `createProfile()`.  This will create a `profile.yaml` in your home directory which stores the location of your R installation and some other information.  It will be filled in automatically, unless your operating system is unrecognized.
```
Comment (jiwalker-usgs) [inline vignettes/vizlab_example.Rmd]: Probably worth commenting that this only needs to be run once, or updated when R is updated.

File: vignettes/vizlab_example.Rmd (hunk 1 above)
Comment (wdwatkins) [inline vignettes/vizlab_example.Rmd]: Once we hit all the MVP milestones?

**Pull request merged**